src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.product-card {
+  width: 200px;
+  padding: 32px 16px 0;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
```

Comment:
Move to the html selector and remove from all other places

Reply:
> Move to the html selector and remove from all other places

Mentor above pointed out to add font-family, and you say to remove it. What is the logic?